engine/src/main/resources/assets/i18n/menu_cs.lang

```diff
@@ -1,316 +1,459 @@
+  "dialog-cancel": "Zrušit",
+  "dialog-no": "Ne",
+  "dialog-ok": "Ok",
+  "dialog-option": "Volba",
+  "dialog-yes": "Ano",
+  "disable-all-modules": "Deaktivovat vše",
+  "disable-launch-popup": "Zapamatovat a již se neptat",
+  "disable-rendering-class": "Vypnout",
+  "discord-presence": "Discord Rich Presence",
+  "download-module": "Stáhnout",
+  "downloading-server-list": "Načítání seznamu serverů ...",
+  "drop-item": "Zahodit předmět",
+  "edit-server": "Upravit",
+  "edit-server-title": "Přidat/Upravit server",
+  "enable-error-reporting": "Odesílat logy selhání na server",
+  "enable-telemetry": "Odesílat telemetrii na server",
+  "enable-rendering-class": "Zapnout",
+  "enter-username-message": "Ahoj a vítej v muliplayeru Terasology!\nPřeješ si aktualizovat tvé jméno?",
+  "enum-editor-prompt": "Vyber hodnotu uzlu:",
+  "enum-editor-title": "Editor výčtových uzlů",
+  "error": "Chyba",
+  "error-credits-not-found": "Soubor Autorů nenalezen!",
+  "error-credits-open": "Chyba při otvírání souboru Autorů!",
+  "error-downloading-server-list": "Chyba při stahování seznamu serverů!",
+  "existing-file-warning": "Soubor %s již existuje. Přeješ si ho přepsat?",
+  "exit-game": "Ukončit Terasology",
+  "experimental": "Experimentální",
+  "extras-label": "Doplňky",
+  "extra-lighting": "Extra osvětlení",
+  "eye-adaption": "Přizpůsobení oka",
+  "failed-to-join": "Chyba při připojení",
+  "file-picker-access-denied-to": "Zamítnut přístup k ",
+  "file-picker-cant-change-dir": "Nepodařilo se změnit složku",
+  "file-picker-file-name": "Název souboru: ",
+  "file-picker-full-path": "Celá cesta k souboru: ",
+  "file-picker-home": "Domvská složka",
+  "file-picker-invalid-file-name": "Neplatný název souboru",
+  "file-picker-invalid-location": "Neplatné umístění",
+  "file-picker-parent": "Nadřazené",
+  "file-picker-root": "Kořenový adresář",
+  "file-picker-roots-title": "Kořenové adresáře souborového systému",
+  "file-picker-title": "Vyber soubor",
+  "flickering-lights": "Blikající světla",
+  "fog": "Mlha",
+  "fov-amount": "Zorné pole",
+  "framerate-limit": "Limit FPS",
+  "game-details-description": "Popis: ",
+  "game-details-title": "Detail hry",
+  "game-details-general-info": "Obecné informace",
+  "game-details-game-title": "Název:",
+  "game-details-game-seed": "Seed:",
+  "game-details-game-duration": "Trvání:",
+  "game-details-modules": "Moduly",
+  "game-details-worlds-and-biomes": "Světy a Biomy",
+  "game-details-worlds": "Světy",
+  "game-details-biomes": "Biomy",
+  "game-details-blocks": "Bloky",
+  "game-details-world-generator": "Generátor světa",
+  "game-details-world-description": "Popis světa",
+  "game-details-no-description": "Bez popisu.",
+  "game-details-version": "Verze:",
+  "game-details-last-play": "Naposledy hráno",
+  "game-details-invalid-module-version-warning": "Varování: manifest.json obahuje nevalidní nebo neexistující verzi modulu. Toto je popisek poslední verze modulu na tvojí classpath.",
```

Comment:
```suggestion
  "game-details-invalid-module-version-warning": "Varování: soubor manifest.json obahuje neplatnou nebo neexistující verzi modulu. Toto je popisek poslední verze modulu na tvojí classpath.",
```
Also, "classpath" sounds wierd

Reply:
That reads better. Not sure about the classpath - it's something that can't be translated. I was actually pretty surprised that we even show something like this to users, I'm really not sure what to do with that.